src/NumericalAlgorithms/FiniteDifference/Wcns5z.hpp

```diff
@@ -27,8 +28,10 @@ class Index;
 namespace fd::reconstruction {
 namespace detail {
 
+namespace {
+// pointwise reconstruction routine for the original Wcns5z scheme
 template <size_t NonlinearWeightExponent>
-struct Wcns5zReconstructor {
+struct Wcns5zWork {
```

Comment:
Not sure this is the best naming or the best way (as a struct) to keep the WENO5Z numerical routine.